### PR TITLE
Add a special 504 message on reruns

### PR DESCRIPTION
**Pull request by mikix**
As a stop-gap measure, we give a special message when receiving a 504 message back from Discovery when making a rerun.

I didn't add a test, because we still don't seem to have a good way to test network interactions. Seems bad, yah?

https://openedx.atlassian.net/browse/DISCO-1548

Screenshot from 2020-01-24 15-00-02

### CODE REVIEW

File: src/data/actions/courseInfo.js [hunk 1]
```diff
@@ -154,7 +154,15 @@ function createCourseRun(courseUuid, courseRunData) {
         return dispatch(push(`/courses/${courseUuid}`));
       })
       .catch((error) => {
-        dispatch(createCourseRunFail(['Course Run create failed, please try again or contact support.'].concat(getErrorMessages(error))));
+        let errorList;
+        if (error.response.status === 504) {
```
Comment (jlajoie) [inline src/data/actions/courseInfo.js]: Might be worth confirming there's a response object here?

**Comment by jlajoie** [comment]
As an aside, should we maybe up the margin in that image between the error box and the buttons? They seem a little snug.

File: src/data/actions/courseInfo.js (hunk 1 above)
Comment (mikix) [reply src/data/actions/courseInfo.js]: Done.

**Comment by mikix** [comment]
> As an aside, should we maybe up the margin in that image between the error box and the buttons? They seem a little snug.

Good point, done.

**Pull request merged**